code/obj/machinery/law_racks.dm

```diff
@@ -602,6 +603,12 @@
 		var/list/mobtextlist = list()
 		for(var/mob/living/M in affected_mobs)
 			mobtextlist += constructName(M, "admin")
+		power_usage = 1000
```

Comment:
It'd be cleaner to do this as a proc like `proc/UpdatePowerUsage()` and have it called here and in `SetLaw()`, that way you don't have to set power usage in the ai law manager datum.